build.gradle

```diff
@@ -33,6 +33,24 @@ ext {
     supportLibraryVersion = "24.2.1"
     butterKnifeVersion = "7.0.1"
     daggerVersion = "2.0.2"
+
+    // POM file
+    GROUP = "com.github.brianPlummer"
```

Comment:
Easy to read, you can verify here: https://mvnrepository.com/artifact/com.github.brianPlummer/tinydancer/0.0.9 and a good example here: https://github.com/square/leakcanary/blob/050b554887cd590c80803c25dcd6c05bdf89eadc/build.gradle#L20.